src/pages/index.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
 import Link from 'gatsby-link'
+import EventTag from '../components/EventTag'
 
 const Home = () => (
   <div>
     <h1>Pride!!!!</h1>
+    <EventTag value="Music" />
```

Comment:
can you mock up a fake tags array that you pass into the component. 

It seems that duplicating the EventTag component when there is going to be indefinite number of tags isn't good practice. 

Pass in an array into EventTag, which then iterates over the array, creating a new EventTagSpan for each item.

Reply:
I think there's a misunderstanding here. I just threw some usages of the EventTag component in the placeholder index page to serve as an example until we get a bit further with feature development.

Once we have the data coming in from contentful, I'm envisioning something like 

```js
props.event.tags.map(tag => <EventTag value={tag} key={tag}>)
``` 

But that would be on the Event page, not here.

Reply:
I just removed this to avoid confusion

Reply:
that's fine!